scripting/get5/recording.sp

```diff
@@ -14,20 +14,27 @@ bool StartRecording() {
   }
 
   char demoName[PLATFORM_MAX_PATH + 1];
+
   if (!FormatCvarString(g_DemoNameFormatCvar, demoName, sizeof(demoName))) {
     LogError("Failed to format demo filename. Please check your demo file format convar.");
     g_DemoFileName = "";
     return false;
   }
 
-  FormatEx(g_DemoFileName, sizeof(g_DemoFileName), "%s.dem", demoName);
+  char demoFolder[PLATFORM_MAX_PATH];
+  char variableSubstitutes[][] = {"{MATCHID}", "{DATE}"};
+  int variableListSize = sizeof(variableSubstitutes);
```

Comment:
You can replace this with inline `2`, or just inline the `sizeof()` call at least. The `variableListSize` variable is redundant.

```c
CheckAndCreateFolderPath(g_DemoPathCvar, variableSubstitutes, 2, demoFolder,
                           sizeof(demoFolder));
```